core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -71,7 +71,8 @@
                       class="conversation-skin-bottom-gadget-panel">
   </oppia-gadget-panel>
 
-  <div ng-if="isOnTerminalCard() && isCurrentCardAtEndOfTranscript()">
+  <div ng-if="isOnTerminalCard() && isCurrentCardAtEndOfTranscript()"
+       class="oppia-exp-summary-container">
```

Comment:
lets put this css-class into conversation_skin_directive.html.

see <style> tag in this file (conversation_skin_directive.html).